templates/Pages/rules_submission.php

```diff
@@ -110,6 +110,16 @@
             "<span class="code">beegfs_ctl</span>"; however, additional
             subdirectories within these directories cannot be precreated.
         </li>
+        <li>
+            Submissions received after the posted deadline for a list may
+            only be accepted upon request to
+            <a href="mailto:committee@io500.org" class="link">committee@io500.org</a>.
+            and only if the result does not affect the winner in any category
+            after the official deadline. In the case where a late submission
+            would be the winner in any category, or is too near or after the
+            final list publication (at the discretion of the committee), then
```

Comment:
How about something like, " In the case where a late submission would be the winner in any category, then a key consideration for acceptance is ensuring there is enough time before publication of the final list.  Any submission not accepted for the current list would be automatically submitted to the following list.